### DISCUSSION TITLE
How to setting the Product/@Version attribute with letter and number

### DISCUSSION THREAD

**Discussion by chancezheng** [wixtoolset/issues#7391] [Questions]
When I set the version attribute with letter, the wix give me the error with the attribute's value is an integer from 0 to 65534

**Comment by robmen** [comment]
That [sounds right](https://learn.microsoft.com/en-us/windows/win32/msi/productversion). What were you expecting?

**Reply by chancezheng** [reply]
such as, Version="1.0.0.P1" , how to support

**Reply by robmen** [reply]
Open a feature request with the Windows Installer team at Microsoft.

**Reply by chancezheng** [reply]
alright, thanks

**Reply by chrpai** [reply]
I remember these battles with management well.  They don't like being told that the Windows Installer platform doesn't support the way they want to do it.  Sorry and best of luck managing up.
